Cart page

Kuchvara opened the pull request.

Kuchvara committed:
first commit

basic functionality

Kuchvara committed:
last edition

Kuchvara committed:
last commint

coupons: discount10, discount20, discount30

may looks little raw

Kuchvara committed:
some bug fixes

Kuchvara committed:
fixes due to checkout page requirements

change functionality due to next task

coupons: discount10, discount20, discount30

vhamolia commented:
Good work!

Need  a few fixes: 
1. Receive an error when add second time same product to cart https://prnt.sc/1yly896
2. Adding to cart doesn't work on product page
3. On product page should be possibility input number of items
4. Please, show stock number on product page
5. Here should be only 2 symbols after dot https://prnt.sc/1ylz3lp  https://prnt.sc/1ylzbcy 
6. Please, add functionality for deleting coupon in the cart
7. Your banks payments make calculations with services, but they shouldn't

Kuchvara committed:
corrections due to tam lead's comments

Kuchvara committed:
coupon cancel fixes

coupon cancel logic fixes due to morning conversation
P.S. additional warranty fixes are on checkout branch
